ocf_data_sampler/numpy_batch/site.py

```diff
@@ -0,0 +1,31 @@
+"""Convert site to Numpy Batch"""
+
+import xarray as xr
+
+
+class SiteBatchKey:
+
+    site = "site"
+    site_capacity_kwp = "site_capacity_kwp"
+    site_time_utc = "site_time_utc"
+    site_t0_idx = "site_t0_idx"
+    site_solar_azimuth = "site_solar_azimuth"
+    site_solar_elevation = "site_solar_elevation"
+    site_id = "site_id"
+    site_latitude = "site_latitude"
+    site_longitude = "site_longitude"
```

Comment:
I don't think any of these are used in convert to numpy batch below. Is there another reason for them to be here? Or do we not use them?

Reply:
thanks, ive remove lat and lon